azext_iot/central/_help.py

```diff
@@ -136,6 +136,51 @@ def _load_central_devices_help():
                 az iot central app device registration-summary --app-id {appid}
         """
 
+    helps[
+        "iot central app device run-command"
+    ] = """
+            type: command
+            short-summary: Run a command on a device and view associated response. Does NOT monitor property updates that the command may perform.
+            long-summary: |
+                Note: payload should be nested under "request".
+                i.e. if your device expects the payload in a shape {"key": "value"}
+                payload should be {"request": {"key": "value"}}.
+                --content can be pointed at a filepath too (.../path/to/payload.json)
+            examples:
+            - name: Run command response
+              text: >
+                az iot central app device run-command
```

Comment:
Are there no connectivity or response timeout arguments to deal with on the central side?

Reply:
Doesn't look like it as of now, its not a well documented concept. Known issue that the team is working on

https://docs.microsoft.com/en-us/rest/api/iotcentral/devices/executecomponentcommand

![image](https://user-images.githubusercontent.com/42591144/86289398-7af10a00-bba0-11ea-87a0-27cb2b79a86a.png)
![image](https://user-images.githubusercontent.com/42591144/86289415-7e849100-bba0-11ea-87af-5479bb213b45.png)

Reply:
#wontfix